liveview-android/src/main/java/org/phoenixframework/liveview/data/dto/ExposedDropDownMenuBoxDTO.kt

```diff
@@ -0,0 +1,118 @@
+package org.phoenixframework.liveview.data.dto
+
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.material3.ExposedDropdownMenuBox
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import org.phoenixframework.liveview.data.core.CoreAttribute
+import org.phoenixframework.liveview.domain.base.ComposableBuilder
+import org.phoenixframework.liveview.domain.base.ComposableView
+import org.phoenixframework.liveview.domain.base.ComposableViewFactory
+import org.phoenixframework.liveview.domain.base.PushEvent
+import org.phoenixframework.liveview.domain.factory.ComposableTreeNode
+import org.phoenixframework.liveview.ui.phx_components.PhxLiveView
+
+/**
+ * Material Design Exposed Dropdown Menu.
+ * Menus display a list of choices on a temporary surface. They appear when users interact with a
+ * button, action, or other control.
+ * Exposed dropdown menus display the currently selected item (most often) in a text field to which
+ * the menu is anchored. If the text field input is used to filter results in the menu, the
+ * component is also known as "autocomplete" or a "combobox".
+ * The first child must be the "anchor", which means the clickable component that will show the
+ * menu. This component must add the `menuAnchor` property. The second component and the following
+ * ones, will be considered menu items. The menu items are commonly `DropDownMenuItem`, but can be
+ * any component.
+ * ```
+ * <ExposedDropDownMenuBox horizontalPadding="16">
+ *   <TextField text={"#{@ddOption}"} readOnly="true" menuAnchor/>
+ *   <DropDownMenuItem phx-click="setDDOption" value="A">
+ *     <Text>Option A</Text>
+ *   </DropDownMenuItem>
+ *   <DropDownMenuItem phx-click="setDDOption" value="B" enabled="false">
+ *     <Text>Option B</Text>
+ *   </DropDownMenuItem>
+ *   <DropDownMenuItem phx-click="setDDOption" value="C">
+ *     <Text>Option C</Text>
+ *   </DropDownMenuItem>
+ * </ExposedDropDownMenuBox>
```

Comment:
Could the structure here match compose more closely? A separate `ComposableView` could be used for the `ExposedDropdownMenu` and have the `ExposedDropdownMenuBox` just be a "dumb" container element.

```heex
<ExposedDropdownMenuBox>
  <TextField ... />
  <ExposedDropdownMenu ...>
    <DropdownMenuItem :for={item <- @items} />
  </ExposedDropdownMenu>
</ExposedDropdownMenuBox>
```

Reply:
I can do that... I actually thought about it, but the compose documentation says:
> Popup which contains content for Exposed Dropdown Menu. Should be used inside the content of ExposedDropdownMenuBox.
Since it's only used in the `ExposedDropdownMenuBox`, I tried to simplify the usage... 
But I'll do that for consistency...

Reply:
I tried to do that, but as you can see in the `ExposedDropdownMenuBoxDTO` file, both `ExposedDropdownMenuBox` and `ExposedDropdownMenu` share the `isExpanded` state. So it will be necessary to pass this parameter to the child composable in order to update it. 
I need more time to investigate a way to architect this, which could impact other files not related to this PR. Could you please open an issue for that?